### DISCUSSION TITLE
How to read past notification messages?

### DISCUSSION THREAD

**Discussion by ninoCan** [LazyVim/LazyVim#1900] [Q&A] [answered]
Sometimes, especially when notifications pop up at start-up, the messages tend to disappear even before I've had time to read them. Is there a way to read past notifications, are they kept somewhere?

**Comment by mstinsky** [comment, marked as answer]
You have a message history in noice.nvim.
You can find the default keybinds here:
https://www.lazyvim.org/keymaps#noicenvim

**Comment by amaanq** [comment]
`:mess` if it's before Noice is loaded

**Reply by blue-fox-maker** [reply]
very useful

**Reply by rjkush17** [reply]
thank-you friend
